src/router/bundle/bundleGenerator.ts

```diff
@@ -129,4 +129,51 @@ export default class BundleGenerator {
         response.entry = entries;
         return response;
     }
+
+    static generateBatchBundle(baseUrl: string, bundleEntryResponses: BatchReadWriteResponse[]) {
```

Comment:
This method is very similar to `generateTransactionBundle`, combine common codes?